src/handlers/motions/motionCreated/handlers/streamingPayments.ts/cancelStreamingPaymentsMotion.ts

```diff
@@ -0,0 +1,35 @@
+import { BigNumber } from 'ethers';
+import { TransactionDescription } from 'ethers/lib/utils';
+import { ContractEvent, motionNameMapping } from '~types';
+import { createMotionInDB } from '../../helpers';
+import {
+  getDomainDatabaseId,
+  getExpenditureDatabaseId,
+  toNumber,
+} from '~utils';
+
+export default async (
+  event: ContractEvent,
+  { name, args: actionArgs }: TransactionDescription,
+  gasEstimate: BigNumber,
+): Promise<void> => {
+  const { args, colonyAddress } = event;
+  const [, , streamingPaymentId] = actionArgs;
+  const [, , domainId] = args;
+
+  if (!colonyAddress) {
+    return;
+  }
+
+  await createMotionInDB(event, {
+    type: motionNameMapping[name],
+    fromDomainId: colonyAddress
```

Comment:
nit-pick: `colonyAddress` is guaranteed to be truthy at this point

Reply:
I shall clean that up! 👌